site/realm/functions/promoteSubmissionToReport.js

```diff
@@ -16,10 +16,14 @@ exports = async (input) => {
     
     const newIncident = {
       title: submission.title,
+      description: submission.description,
       incident_id: lastIncident.incident_id + 1,
       reports: [],
       editors: ["Sean McGregor"],
       date: submission.incident_date,
+      "Alleged deployer of AI system": !Array.isArray(submission.deployers) ? [] : submission.deployers,
+      "Alleged developer of AI system": !Array.isArray(submission.developers) ? [] : submission.developers,
+      "Alleged harmed or nearly harmed parties": !Array.isArray(submission.harmed_parties) ? [] : submission.harmed_parties,
```

Comment:
Is this just checking that the field exists? Or is it possible for non-array values to end up here? If the former, why not just write `submission.deployers || []`? If the latter, is it appropriate to just use an empty array, or would it be better to add handling for other possible types? e.g. `typeof submission.deployers === 'string' ? [submission.deployers] : []`

Reply:
@lmcnulty you're right. Actually it should always be an array, so I'm leaving it as `submission.deployers || []`